android/assets/jsons/Civ V - Gods & Kings/Buildings.json

```diff
@@ -840,9 +840,10 @@
 		"culture": 3,
 		"cityStrength": 12,
 		"isWonder": true,
+        "greatPersonPoints": {"Great Scientist": 1},
 		"uniques": ["[+25]% City Strength from defensive buildings"],
 		"requiredTech": "Metallurgy",
-		"quote": "'The Kremlin is constantly changing the rules of the game to suit its purposes. We are not playing chess, we're playing roulette.' - Garry Kasparov"
+		"quote": "'The Law is a fortress on a hill that armies cannot take or floods wash away.' - The Prophet Muhammad"
```

Comment:
Though, if we do the quote change, we should likely do it in Vanilla too.

``` suggestion
		"quote": "'The Kremlin is constantly changing the rules of the game to suit its purposes. We are not playing chess, we're playing roulette.' - Garry Kasparov"
```

Reply:
Actually: Please don't revert changes a contributor made with well-explained arguments unless you can trump those arguments.

And Rob: You do know Studio: Select some lines: Right-click -> Git -> "Show History for Selection..." is the best visualization of git blame you can get? Right, I didn't think to use it when I first read this.